README.md

```diff
@@ -1,99 +1,200 @@
 
-:bust_in_silhouette: **Joy Phoebe**
-- GitHub: [@githubhandle](https://github.com/joyapisi)
-- Twitter: [@twitterhandle](https://twitter.com/joyphoebe300)
-- LinkedIn: [LinkedIn](https://www.linkedin.com/in/joyapisi/)
+If you think that change is necessary - open a Pull Request in this repository and let your code reviewer know about it. 4. Run `npx stylelint "**/*.{css,scss}"` on the root of your directory of your project.
+
+5. Fix linter errors.
+
+6. **IMPORTANT NOTE:** feel free to research auto-correct options for Stylelint if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## :telescope: Future Features <a name="future-features"></a>
-  - Practice more Ruby syntax
+## 👥 Authors <a name="authors"></a>
+
+👤 **Joy Phoebe**
+
+- GitHub: (https://github.com/joyapisi)
+- Twitter: (https://twitter.com/joyapisi)
+- LinkedIn: (https://linkedin.com/in/joy-phoebe-00b80a13a)
+
+## 🤝 Contributing <a name="contributing"></a>
+
+- GitHub: (https://github.com/Shaheryar0054)
+
+## 🔭 Future Features <a name="future-features"></a>
 
+# **[Feature-1]**
+
+- Add more styling (aesthetics)
+
+# **[Feature-2]**
+
+- Add mobile version
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## :handshake: Contributing <a name="contributing"></a>
 Contributions, issues, and feature requests are welcome!
-Feel free to check the [issues page](https://github.com/newhoteng/Enumerable/issues).
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+Feel free to check the [issues page](https://github.com/joyapisi/To-Do-List/issues).
 
-## :star: Show your support <a name="support"></a>
-If you like this project give it a star.
+## ⭐️ Show your support <a name="support"></a>
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+If you like this project, kindly leave a comment below and share it with someone who enjoys coding! Coding is all about continuous learning and allowing yourself to be a beginner. Keep going!
+
+## 🙏 Acknowledgments <a name="Microverse Inc."></a>
+
+I'm thankful to Microverse for providing a study platform which guided me through this project.
 
-## :pray: Acknowledgements <a name="acknowledgements"></a>
-- Thanks to Microverse for encouraging us to do this project.
+## ❓ FAQ <a name="faq"></a>
+
+- **[Question_1]**
+  An easier and quicker way to understand webpack?
+
+  - This YouTube<a href="https://www.youtube.com/watch?v=MpGLUVbqoYQ">video</a> will help you understand webpack better and in details
+
+- **[Question_2]**
+  Where can I download node JS for installation?
+
+  - Node Js can be downloaded here- <ul>
+    <li><a href="https://nodejs.org/en/download/"> Node JS </a></li>
+  </ul>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.txt) licensed.

```

Comment:
- [ ]  Good job work on project documentation as it was suggested, but you MIT link is empty. please fix it as well and make it work properly to have a professional project documentation.